airflow/models/dagrun.py

```diff
@@ -138,7 +138,7 @@ def __init__(
         self,
         dag_id: Optional[str] = None,
         run_id: Optional[str] = None,
-        queued_at: Optional[datetime] = __NO_VALUE,
+        queued_at: Optional[datetime] = __NO_VALUE,  # type: ignore
```

Comment:
This can benefit from the new `ArgNotSet` class in `airflow.utils.types`.

Reply:
Nice. fixed in f77d88edf